test/spec/ol/format/wmtscapabilities.test.js

```diff
@@ -115,4 +115,111 @@ describe('ol.format.WMTSCapabilities', function() {
     });
 
   });
+
+  describe('when parsing ign.xml', function() {
+
+    var parser = new ol.format.WMTSCapabilities();
+    var capabilities;
+    before(function(done) {
+      afterLoadText('spec/ol/format/wmts/ign.xml', function(xml) {
+        try {
+          capabilities = parser.read(xml);
+        } catch (e) {
+          done(e);
+        }
+        done();
+      });
+    });
+
+    it('can read Capability.Contents.Layer', function() {
+
+      expect(capabilities.Contents.Layer).to.be.an('array');
+      expect(capabilities.Contents.Layer).to.have.length(1);
+
+
+      var layer = capabilities.Contents.Layer[0];
+      expect(layer.Abstract).to.be
+        .eql('Photographies aériennes');
+      expect(layer.Identifier).to.be.eql('ORTHOIMAGERY.ORTHOPHOTOS');
+      expect(layer.Title).to.be.eql('Photographies aériennes');
+
+      expect(layer.Format).to.be.an('array');
+      expect(layer.Format).to.have.length(1);
+      expect(layer.Format[0]).to.be.eql('image/jpeg');
+
+      expect(layer.Style).to.be.an('array');
+      expect(layer.Style).to.have.length(1);
+      expect(layer.Style[0].Identifier).to.be.eql('normal');
+      expect(layer.Style[0].isDefault).to.be(true);
+      expect(layer.Style[0].Title).to.be.eql('Données Brutes');
+      expect(layer.Style[0].LegendURL[0].href).to.be
+        .eql('http://www.geoportail.gouv.fr/depot/LEGEND.jpg');
+      expect(layer.Style[0].LegendURL[0].format).to.be.eql('image/jpeg');
+
```

Comment:
it would be okay for me if this test case focuses only on the TileMatrixSet stuff, no need to test other element here IMHO. But not really important.